Description: The usage of this method requires an unreleased version of the
  golang client. It is a trivial method, so I am just inlining the
  implementation.

--- a/cmd/prometheus/main.go
+++ b/cmd/prometheus/main.go
@@ -583,7 +583,8 @@
 	defer func() {
 		if err == nil {
 			configSuccess.Set(1)
-			configSuccessTime.SetToCurrentTime()
+			//configSuccessTime.SetToCurrentTime()
+			configSuccessTime.Set(float64(time.Now().UnixNano()) / 1e9)
 		} else {
 			configSuccess.Set(0)
 		}
